lenny/configs/__init__.py

```diff
@@ -29,6 +29,8 @@
 READIUM_PORT = int(os.environ.get('READIUM_PORT', 15080))
 READIUM_BASE_URL = f"http://lenny_readium:{READIUM_PORT}"
 
+OTP_KEY = os.environ.get('OTP_KEY')
```

Comment:
This is exactly right for our testing but we won't want to check this change in

We **will** want to import the LENNY SEED because that will be used in our production hash.